code/modules/asset_cache/asset_list.dm

```diff
@@ -139,6 +139,13 @@ GLOBAL_LIST_EMPTY(asset_datums)
 	/// Defaults to false so we can process this stuff nicely
 	var/load_immediately = FALSE
 
+#ifdef DO_NOT_DEFER_ASSETS
+/datum/asset/spritesheet/New()
+	// ALWAYS load now
+	load_immediately = TRUE
```

Comment:
Better to `#ifdef` the var declaration

Reply:
Like:

```dm
#ifdef DO_NOT_DEFER_ASSETS
    var/load_immediately = TRUE
#else
    var/load_immediately = FALSE
#endif
```

Reply:
what if a subtype overrides it, or double overrides it? or what if someone sets it manually off their new()

Reply:
As in where you define load_immediately, wrap it in an ifdef

Reply:
no I get that but isn't that still vulnerable to people overriding it to FALSE by accident of OOP, or setting it manually somewhere in New()?

Reply:
Oh I guess

What you have here feels kinda hacky though, in my head I'm thinking maybe even

```dm
/datum/asset/spritesheet/proc/should_load_immediately()
#ifdef DO_NOT_DEFER_ASSETS
    return TRUE
#else
    return load_immediately
#endif
```

Reply:
hm. ok yeah sure